api/proto/info/info.proto

```diff
@@ -427,20 +427,24 @@ message SmartAttr {
   uint64 worst = 3;   // worst value that Value has ever reached
   uint64 thresh = 4;  // the value that the Value of the same attribute must reach for the attribute's state to be considered critical.
   string when_failed = 5; // critical state if no empty
+  uint64 raw_value = 6;
 }
 
 message SmartMetric {
   SmartAttr reallocated_sector_ct = 1;
   SmartAttr power_on_hours = 2;
   SmartAttr power_cycle_count = 3;
-  SmartAttr reallocated_event_count = 4;
+  SmartAttr reallocated_event_count = 4; // Deprecated
```

Comment:
Why deprecate? This is supposedly useful on rotating disks (don't know about SSDs).

Reply:
In testing, I did not receive this option, it is not always available, of course, I can leave it, but there is no guarantee that it will be filled.

Reply:
If it is useful then it makes sense to include it in the protobuf and then if some devices do not support it have a convention e.g., that we set the value in the API to -1 to indicate "not available".